### ISSUE TITLE
PAYDAY The Heist displays black screen during first start (24240)

### ISSUE THREAD

**Issue by pprotonrep** [ValveSoftware/Proton#324]
The game [PAYDAY The Heist](https://store.steampowered.com/app/24240/) displays a black screen when started for the first time.

Modifying `compatdata/24240/pfx/drive_c/users/steamuser/Local Settings/Application Data/PAYDAY/renderer_settings.xml` so that the `resolution` key contains the native resolution (in my case from `1280 800` to `1920 1200`) allowed the game to start without problems. Gameplay seems nice and smooth too (just played for a couple of minutes).

- Ubuntu 18.04.1 LTS (64 bit 4.15.0-33-generic)
- GeForce GTX 1070/PCIe/SSE2 (396.54)
- Proton 3.7 (default configuration)

[System information from steam](https://gist.github.com/pprotonrep/c67cbb6f56d9b88af24952fa45652cdf)

**Comment by pprotonrep** [comment]
FWIY here similar issues related to default resolution: https://github.com/ValveSoftware/Proton/issues/132 https://github.com/ValveSoftware/Proton/issues/141 https://github.com/ValveSoftware/Proton/issues/164

**Comment by vanyasem** [comment]
I was able to launch the game with the default resolution, and then change it to 4k in game's settings without any issues.

System info: https://gist.github.com/vanyasem/f787469ec8c50cdd3728340ef4c73269
![2018-09-09-164011_3470x1000_scrot](https://user-images.githubusercontent.com/13350458/45265100-1c98eb80-b44f-11e8-9452-98e5c4a885ee.png)

**Comment by YellowOnion** [comment]
I'm getting severe FPS drops after about 30mins of game play, and there's graphical glitches now too.

**Comment by LingMan** [comment]
This game has been whitelisted to use Proton 3.7 by default. Unfortunately the game fails to launch for me with Proton 3.7-8. After the initial "ESRB content rating" video the game just closes. Doesn't even get to the main menu. That problem does not appear with Proton 3.16-9 or newer.

The graphical glitches @YellowOnion mentioned above only happen with Proton 5.0+, and are being tracked on the dxvk side (https://github.com/doitsujin/dxvk/issues/2255). You can workaround them by switching to WineD3D with:
`PROTON_USE_WINED3D=1 %command%`

@kisak-valve: Could this game's whitelist be updated to any of Proton {3.16, 4.2, 4.11}? Those launch fine and don't have graphical glitches.

@YellowOnion: Might be useful to know if that FPS drop you mentioned also happens with the WineD3D workaround.

**Comment by kisak-valve** [comment]
@pprotonrep, can you test if one of those Proton versions help you as well?

**Comment by YellowOnion** [comment]
> @YellowOnion: Might be useful to know if that FPS drop you mentioned also happens with the WineD3D workaround.

After a quick google yesterday I noticed that Payday 2 on Windows has a very similar issue maybe related to Ryzen and threading problems: https://www.youtube.com/watch?v=SsaU-6QOpCw sadly the work around doesn't work on my machine.

I also noticed that the FPS is fine if I don't use my mouse or keyboard.

I'll try WineD3D later, though I suspect it's not graphics related as per above reasons.

I'll try fiddle with mouse polling rate to see if it's at all helps.
